executor/executor_test.go

```diff
@@ -4395,6 +4395,7 @@ func (s *testSplitTable) TestClusterIndexSplitTableIntegration(c *C) {
 func (s *testSplitTable) TestClusterIndexShowTableRegion(c *C) {
 	tk := testkit.NewTestKit(c, s.store)
 	atomic.StoreUint32(&ddl.EnableSplitTableRegion, 1)
+	tk.MustExec("set global tidb_scatter_region = 1")
```

Comment:
Will it affect other test cases? How about defer setting it back?

Reply:
The test suite `testSplitTable` is serial and expected `tidb_scatter_region == 1`, it won't affect other test cases.

Reply:
should use defer set it back to 0? otherwise it will take effect int following Testxxx in `testSplitTable Suite` or that's what you exactly want.

Reply:
It will take effect on the following tests. But it does not matter.